Wikipedia/Code/WMFLanguageCell.h

```diff
@@ -1,12 +1,14 @@
+#import "WMFTableViewCellWithCustomMinusButton.h"
 
 /// Table cell designed for displaying a language autonym & a page title in that language.
-@interface WMFLanguageCell : UITableViewCell
+@interface WMFLanguageCell : WMFTableViewCellWithCustomMinusButton
```

Comment:
WMFTableViewCellWithCustomDeleteButton - minor, but minus happens to be the symbol that is used, but delete is more to the point.